docs/docs/Configuration.md

```diff
@@ -53,9 +104,12 @@ fsharp_strict_mode=false
 Please note that you should only add settings to the `.editorconfig` file when you want to deviate from the default settings.
 Copying the entire list above is unnecessary.
 
-### indent_size
+# Each setting explained
+
+<a id="indent_size"></a>
+## indent\_size
```

Comment:
I'm on the fence about this one. Escaping work and appears to be legal Markdown, however, it is a missed opportunity to tackle https://github.com/fsprojects/FSharp.Formatting/issues/389.

Reply:
@nojaf I had a quick look at that and that should be solved, sure, but I lack the time and don't know enough (yet) of that lib. Also, if I'm not going to fix it, the bug has been out there for 6 years... hence this approach.